custom-recipes/timeseries-forecast-1-train-evaluate/recipe.py

```diff
@@ -79,13 +78,13 @@
 
 logger.info("Completed training session {} in {:.2f} seconds".format(session_name, perf_counter() - start))
 
-params["evaluation_dataset"].write_with_schema(training_session.get_evaluation_metrics_df())
-evaluation_results_columns_descriptions = {**METRICS_COLUMNS_DESCRIPTIONS, **EVALUATION_METRICS_DESCRIPTIONS}
-set_column_description(params["evaluation_dataset"], evaluation_results_columns_descriptions, to_lowercase=True)
+params["evaluation_dataset"].write_with_schema(training_session.get_evaluation_metrics_df(), dropAndCreate=params["no_partitioning"])
+evaluation_results_columns_descriptions = training_session.create_evaluation_results_columns_descriptions()
```

Comment:
@StanislasGuinel is that still current? This method doesn't exist in release/1.0

Reply:
yes the create_evaluation_results_columns_descriptions method should be used, it's mostly to lowercase metrics like "MSE"